flake8diff/flake8.py

```diff
@@ -147,6 +152,22 @@ def get_color_kwargs(self, details):
             header=self.color_getter('header')(get('header')),
         )
 
+    def should_include_violation(self, violation, changed_lines):
+        """
+        Check if given violation should be included.
+
+        This method accounts for strictness modes.
+        """
+        strict_mode = self.options.get('strict_mode', 'only_lines')
+        if strict_mode == 'only_lines':
+            return violation['line_number'] in changed_lines
+        elif strict_mode == 'file':
+            return True
+        else:
+            msg = 'Strict mode {} is not supportec'.format(strict_mode)
```

Comment:
Its already checked in [`main.py`](https://github.com/dealertrack/flake8-diff/pull/22/files#diff-cc426425bfd8b2a2ed002c53fd38cf9eR108). This is just in case..

sorry for the typo. will fix